pilot/pkg/xds/endpoint_builder.go

```diff
@@ -88,6 +90,8 @@ func NewEndpointBuilder(clusterName string, proxy *model.Proxy, push *model.Push
 		port:       port,
 	}
 
+	b.populateFailoverPriorityLabels()
```

Comment:
nit: I would just inline this. Making it a method makes it seem like it can be called at arbitrary times - it must only be called in NewEndpointBuilder